packages/fela-bindings/src/ThemeProviderFactory.js

```diff
@@ -5,26 +5,20 @@ export default function ThemeProviderFactory(
   BaseComponent: any,
```

Comment:
`baseComponent` and `the objectEach` import don't seem to be needed an more

Reply:
Fair enough, I will remove it before pushing. This is kind of a hot fix as I need this functionality now :D